docs/rules/extensions.md

```diff
@@ -120,7 +122,9 @@ import bar from './bar.json';
 
 import Component from './Component.jsx';
 
-import * as path from 'path';
```

Comment:
why is this one removed?

Reply:
@ljharb I've restored it, please have a look.